app/videobridge/server/methods/jitsiSetTimeout.js

```diff
@@ -50,9 +50,7 @@ Meteor.methods({
 
 			return jitsiTimeout || nextTimeOut;
 		} catch (error) {
-			SystemLogger.error('Error starting video call:', error);
-
-			throw new Meteor.Error('error-starting-video-call', error.message);
+			SystemLogger.error('Error starting video call:', error.message);
```

Comment:
I'm ok with this particular change, but I'm with Kevin on removing the `Meteor.Error`, we should not remove it.. if the UI is not showing the error, that's a bug that should be fixed on frontend.. but if we remove the `Meteor.Error` will be breaking the API, as it will start returning success instead of an error.